root/static/scripts/edit/URLCleanup.js

```diff
@@ -2175,6 +2175,26 @@ const CLEANUPS = {
       return {result: false};
     },
   },
+  'offiziellecharts': {
+    match: [new RegExp('^(https?://)?([^/]+\\.)?offiziellecharts\\.de/', 'i')],
+    type: LINK_TYPES.otherdatabases,
+    clean: function (url) {
+      return url.replace(/^(?:https?:\/\/)?(?:[^\/]+\.)?offiziellecharts\.de\//, 'https://www.offiziellecharts.de/');
```

Comment:
Sure, why not :) I didn't see any but there might be some. Added.

Reply:
Only 5 URLs have been entered so far. I gave an example of actual fragment from the website. Query parameters should always be cleared if possible because many websites add their own `ref=` or alike to external links.

Reply:
Sure! I meant I tried to click some links in the site and didn't get weird URLs, so I kinda assumed there were none. You're better at it than I am! :smile: